campgns/origplus/map00012.txt

```diff
@@ -116,7 +116,7 @@ ENDIF
 
 IF(PLAYER1,DUNGEON_DESTROYED == 1)
 	IF(PLAYER2,DUNGEON_DESTROYED == 1)
-	SET_FLAG(PLAYER0,FLAG1,1)
+	SET_FLAG(PLAYER0,FLAG7,1)
```

Comment:
Are you sure about this? FLAG1 does indeed seem to be incorrect, but it looks like the line needs to be removed completely here. I think it was intended there to make sure it would not trigger after the fact, but it seems to have been done another way.
Line 19 of the script _does_ have this error though.